### PR TITLE
[testing] [pre-rfc] [wip] split spread in travis

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
We can go ahead and then remove the linode: line

File: run-checks [hunk 1]
```diff
@@ -249,7 +249,11 @@ if [ "$SPREAD" = 1 ]; then
     export PATH=$TMP_SPREAD:$PATH
     ( cd "$TMP_SPREAD" && curl -s -O https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz && tar xzvf spread-amd64.tar.gz )
 
-    spread google: linode:
+    if [ "${SPREAD_SYSTEM:-}" ]; then
+        spread "$SPREAD_SYSTEM"
+    else
+        spread google: linode:
```
[the remark this answers is not in the document]
Comment (chipaca) [reply run-checks]: Yep. I left it, but it has no effect really

**Comment by chipaca** [comment]
@sergiocazzolato it's good to see the breakdown, and it's really nice to be able to redo only a part of the spread tests. But, it's significantly slower (about 2x), because it only runs ~5 at a time.

Maybe we want a hybrid approach?

**Comment by chipaca** [comment]
I'm going to declare this experiment finished, and the conclusion is: don't do it. It would work well if we only ever did one PR at a time, but it blocks progress in any other PR while this one is being tested.

**Comment by bboozzoo** [comment]
Some past experiments #4391 #4393. IIRC we expected some issues with job queue accounting, probably something about hitting job count limit 2-3 times in a single PR instead of having it exhausted by 2-3 PRs as it is now.

**Pull request closed without merging**